releases/release-5.3.1.md

```diff
@@ -0,0 +1,215 @@
+---
+title: TiDB 5.3.1 Release Notes
+category: Releases
+---
+
+# TiDB 5.3.1 Release Notes
+
+Release Date: xx, 2022
+
+TiDB version: 5.3.1
+
+## Compatibility changes
+
+TiDB
+
+TiKV
+
+PD
+
+TiDB Dashboard
+
+TiFlash
+
+Tools
+
+    - Backup & Restore (BR)
+
+    - TiCDC
+
+    - Dumpling
+
+    - TiDB Binlog
+
+    - TiDB Lightning
+
+## Feature enhancements
+
+## __unsorted
+
++ TiDB
```

Comment:
tidb bug fix:
    - Fix date formate identifies '\n' as invalid separator [#32503](https://github.com/pingcap/tidb/pull/32503)
    - Fix `alter column set default` wrongly updates the schema [#31074](https://github.com/pingcap/tidb/issues/31074)
    - Fix a bug that turning on tidb_restricted_read_only won't automatically turn on tidb_super_read_only [#31745](https://github.com/pingcap/tidb/issues/31745)
    - Fix greatest and least function with collation get wrong result [#31789](https://github.com/pingcap/tidb/issues/31789)
    - Fix the crash or error when generating an empty mpp task list. [#31636](https://github.com/pingcap/tidb/issues/31636)
    - Fix index join bug caused by innerWorker panic [#31494](https://github.com/pingcap/tidb/issues/31494)
    - Fix double column value are different with MySQL after changing column type from float to double [#31372](https://github.com/pingcap/tidb/issues/31372)
    - Fix a data race that may cause "invalid transaction" error when executing a query using index lookup join. [#30468](https://github.com/pingcap/tidb/issues/30468)
    (dup) - Fix the issue that executing the `INSERT ... SELECT ... ON DUPLICATE KEY UPDATE` statement gets panic [#28078](https://github.com/pingcap/tidb/issues/28078)
    - Fix a bug when reducing order by clause for the index which leads to the wrong result. [#30271](https://github.com/pingcap/tidb/issues/30271)
    - The TiDB server now maps a user to an entry in the mysql.user table more consistently. [#30450](https://github.com/pingcap/tidb/pull/30450)
    - Fix `MaxDays` and `MaxBackups` not working for slow log. [#30172](https://github.com/pingcap/tidb/pull/30172)

lighting unsorted:
    - Fix the bug that lightning may not clean up metadata schema when some of the import contains no source files. [#28144](https://github.com/pingcap/tidb/issues/28144)
    - Fix the bug that lighting return error if gcs url starts with gs:// [#32591](https://github.com/pingcap/tidb/issues/32591)
    - Make tidb-lightning pre-check output message clearer [#30395](https://github.com/pingcap/tidb/issues/30395)
    - Avoid tikv trigger auto region split by lower the ingest kv count threshold [#30018](https://github.com/pingcap/tidb/issues/30018)
    (dup) - Fix the issue that TiDB Lightning does not report errors when the S3 storage path does not exist [#28031](https://github.com/pingcap/tidb/issues/28031) [#30709](https://github.com/pingcap/tidb/issues/30709)
    - Lightning: fix log doesn't output to stdout when passing `--log-file="-"` [#29876](https://github.com/pingcap/tidb/issues/29876)

br unsorted:
    - Fix a bug that caused region unbalanced after restoring. [#31034](https://github.com/pingcap/tidb/issues/31034)